packages/core/docs/reference.md

```diff
@@ -1,3 +1,11 @@
 # Configuration reference
 
-- [Core config](recipes/core.md)
+- [Core](reference/core.md)
+- Service
+  - [Plugins](reference/service/plugins.md)
+  - [HTTP](reference/service/http.md)
+  - [AMQP](reference/service/amqp.md)
+  - [Logger](reference/service/logger.md)
+  - [Router](reference/service/router.md)
+  - [Validator](reference/service/validator.md)
```

Comment:
Would be better if we move this section to plugins. 
- service
   - plugins
      - http
      - amqp

Reply:
@BrRenat My intention was to make the reference structure to reflect the config structure. So the `core` is the root level of the config, and the `plugins` are the next one. `plugins` is the config section as well.
How about that?
![image](https://user-images.githubusercontent.com/2344246/68669920-99d45a80-0543-11ea-9459-21148bf07ffc.png)

Reply:
@BelkinaDasha, I think separate config structure levels a complicates and confuses. It would be better to define configs on one level.